src/Interpreters/InterpreterSystemQuery.cpp

```diff
@@ -479,11 +482,17 @@ BlockIO InterpreterSystemQuery::execute()
             getContext()->checkAccess(AccessType::SYSTEM_THREAD_FUZZER);
             ThreadFuzzer::start();
             break;
+        case Type::UNFREEZE:
+        {
+            getContext()->checkAccess(AccessType::SYSTEM_UNFREEZE);
+            result = Unfreezer().unfreeze(query.backup_name, getContext());
```

Comment:
This is the only one place in this method where some result returned. I thing here should be a comment with explanation, why.